sftp-common.c

```diff
@@ -245,6 +249,16 @@ ls_file(const char *name, const struct stat *st, int remote, int si_units)
 		tbuf[0] = '\0';
 	ulen = MAX(strlen(user), 8);
 	glen = MAX(strlen(group), 8);
+
+#ifdef IRONCORE
+	/*  If file has .iron extension, prefix with lock icon  */
+	int offset = iron_extension_offset(name);
+	char nbuf[512];
+	char * icon_prefix = (offset > 0) ? IRON_LOCK_ICON : "  ";
```

Comment:
Most other places you allow the file name to consist of only the iron extension. Here you don't. Merely pointing out an inconsistency.